src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php

```diff
@@ -277,6 +281,10 @@ public function getValidationRuleForUpdate(): ParamRuleCollection
                 self::PARAMETER_DIRECTORY,
                 new Rule(Rules::BOOL_TYPE),
             ),
+            new ParamRule(
```

Comment:
is it possible for me to send without `claim` parameter. We have to keep backward compatible.
@ManulMax check this. I have added that to the ticket as well
Basically `claim` param should be optional and it should have a default value maybe `false`